R/decimal-dates.r

```diff
@@ -48,16 +47,12 @@ decimal_date.its <- function(date)
 #' date <- ymd("2009-02-10")
 #' decimal <- decimal_date(date)  # 2009.11
 #' date_decimal(decimal) # "2009-02-10 UTC"
-date_decimal <- function(decimal, tz = NULL) {
+date_decimal <- function(decimal, tz = "UTC") {
   Y <- trunc(decimal)
   ## parsing is much faster than updating
-  start <- make_datetime(Y, 1L, 1L)
-  end <- make_datetime(Y + 1L, 1L, 1L)
+  start <- make_datetime(Y, 1L, 1L, tz = tz)
+  end <- make_datetime(Y + 1L, 1L, 1L, tz = tz)
   seconds <- as.numeric(difftime(end, start, units = "secs"))
   frac <- decimal - Y
   end <- start + seconds*frac
-  if (!is.null(tz))
```

Comment:
Nevermind. That removal is the main point of the RP. Could you please explicitly return the value? Now the return value is invisible.  Thanks.